src/firefox/index.js

```diff
@@ -88,7 +88,7 @@ export type FirefoxRunnerParams = {|
   'no-remote'?: boolean,
   'foreground'?: boolean,
   'listen': number,
-  'binary-args'?: Array<string> | string,
+  'args'?: Array<string> | string,
```

Comment:
You cannot simply rename `binary-args` to `args` here, because this is the API to an external library: https://github.com/mozilla-jetpack/node-fx-runner/blob/71e3848b6adac1054829391c826fd88cfb28f621/lib/run.js#L42-L48

Revert all changes in this file (and use `binaryArgs` instead of `args: binaryArgs` in the other file).

Reply:
Ok, so just keep args only for cli? all other code will remain as is?

Reply:
Yes. The external interface of `web-ext run` will use `args`, and `binary-args` is only used internally, imposed by the contract of the fx-runner interface.